### ISSUE TITLE
Multi-Cursor Strange insert after deleting with Ctrl-Backspace

### ISSUE THREAD

**Issue by apiep** [VSCodeVim/Vim#1224]
Please *thumbs-up* 👍 this issue if it personally affects you! You can do this by clicking on the emoji-face on the top right of this post. Issues with more thumbs-up will be prioritized.

-----

### What did you do?
select `very` word with multi cursor I use `gc`, then change the words with `c`
```
a [very] long line
a [very] long line
a [very] long line
```
type some words, then delete it with Ctrl-Backspace, then type some character.

### What did you expect to happen?
It will place the character in current cursor position

### What happened instead?
The character will not be placed in the cursor position, instead it will placed on cursor position + how many character deleted.
Like this:
```
a longa line
a longa line
a longa line
```

### Technical details:

* VSCode Version: 1.8.1
* VsCodeVim Version: 0.4.10
* OS: Windows 10 x64

**Comment by johnfn** [comment]
Oh yeah, good one. We register backspace but not ctrl-backspace.

**Comment by apiep** [comment]
Thanks 😁

**Comment by johnfn** [comment]
(The fix until we get around to fixing this is to simply switch back into normal mode and use db or something.)

**Comment by xconverge** [comment]
fixed in #1434 on master
